Hyperledger_Fabric_Technical_Charter.md

```diff
@@ -1,116 +1,169 @@
+    1.  The Technical Steering Committee (the “TSC”), also referred to as the Ecosystem TSC, will be responsible for facilitating the governance and providing overarching technical coordination for the open source Project and its ecosystem of **Sub-Projects**. Its decisions on key ecosystem-wide matters are made according to the voting procedures in Section 5.
 
-   6. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC.  The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and Hyperledger Foundation, a directed fund of The Linux Foundation.
+    2.  **TSC Composition:** The TSC voting members are initially those **Sub-Project Maintainers** from certain foundational **Sub-Projects**. Initial TSC voting members are those listed in the MAINTAINERS file in the `fabric` repository ([https://github.com/hyperledger/fabric/blob/main/MAINTAINERS.md](https://github.com/hyperledger/fabric/blob/main/MAINTAINERS.md)) and those in the `fabric-gateway` repository ([https://github.com/hyperledger/fabric-gateway/blob/main/MAINTAINERS.md](https://github.com/hyperledger/fabric-gateway/blob/main/MAINTAINERS.md)). Once a new **Sub-Project** is approved (as per Section 5) and its required repositories are created, the TSC voting membership can be expanded to include designated **Sub-Project Maintainers** from this new **Sub-Project**. The initial inclusion of maintainers from a newly recognized **Sub-Project** may be approved by an existing TSC member. For subsequent changes to a **Sub-Project's** representation on the TSC, for example adding all maintainers of a new repository within a Sub-Project to the TSC, a simple majority approval from the respective existing TSC from the specified Sub-Project is required. The TSC may also, by a decision made under the terms of Section 5, choose an alternative approach for determining the voting members of the TSC, and any such alternative approach (together with a list of then-current TSC voting members) will be documented in the MAINTAINERS files or a similar publicly accessible document. Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
 
-   7. Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
+    3.  **Roles within the Ecosystem:** The Project ecosystem generally will involve Contributors and **Sub-Project Maintainers**. The TSC may adopt or modify roles so long as the roles are documented by the TSC. Unless otherwise documented:
```

Comment:
My reading of if the original section is that it describes expected roles within any repository governed by the TSC. I am not sure it really needs to change. It certainly seems less clear to me now. Assuming my interpretation of the original section is correct, maybe the wording can just be lightly modified to clarify that it relates to repository roles?